scripts/setup-services.sh

```diff
@@ -24,15 +24,19 @@ apt-get update -y
 apt-get install -y gcc-4.8 g++-4.8
 update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-4.8 60 --slave /usr/bin/g++ g++ /usr/bin/g++-4.8
 # ---
+# -- Installing Yarn
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+#--
 apt-get install -y git # For id.webmaker.org
-npm install -g http-server --loglevel=error # For hosting published projects
-npm install -g node-gyp --loglevel=error # For bcrypt
+yarn global add http-server --loglevel=error # For hosting published projects
```

Comment:
for this it looks like the command is yarn run
https://github.com/yarnpkg/yarn/issues/1523